src/winetricks

```diff
@@ -11971,6 +11971,60 @@ __EOF__
 
 #----------------------------------------------------------------
 
+w_metadata office2013pro apps \
+    title="Microsoft Office 2013 Professional" \
+    publisher="Microsoft" \
+    year="2013" \
+    media="cd" \
+    file1="setup.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Microsoft Office/Office15/WINWORD.EXE"
+
+load_office2013pro()
+{
+    if [ "$W_ARCH" = "win64" ]; then
+        w_die "Installation of Office 2013 Pro will not succeed on a 64bit wine prefix"
+    fi
```

Comment:
Use w_package_broken_win64

Reply:
agreed. 
would `w_package_unsupported_win64` be better?